tests/conftest.py

```diff
@@ -1,19 +1,14 @@
 import atexit
 import os
 import sys
-from distutils.version import StrictVersion
```

Comment:
distutils were [removed in 3.12.](https://github.com/nodejs/node-gyp/issues/2869). I'm not sure how tests ever passed there